content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -521,3 +519,19 @@ Interactive transaction error:
 - Could not parse interactive transaction ID
 - Could not find Query Engine for the specified host and transaction ID
 - Could not start interactive transaction
+
+### Accelerate
+
+As Accelerate connections operate differently from local database connections, error codes are different from regular ones. In general, all Accelerate-related errors start with `P6xxx`.
+
+#### <inlinecode>P6004</inlinecode>
+
+**QueryTimeout**—The global timeout of Accelerate has been exceeded. You can find the limit [here](/data-platform/accelerate/limitations#query-timeout-limit).
+
+#### <inlinecode>P6005</inlinecode>
+
+**InvalidParameters**—The user supplied invalid parameters. Currently only relevant for transaction methods. For example, setting a timeout that is too high. You can find the limit [here](/data-platform/accelerate/limitations#interactive-transactions-query-timeout-limit).
+
+#### <inlinecode>P6009</inlinecode>
+
+**ResponseSizeLimitExceeded**—The global response size limit of Accelerate has been exceeded. You can find the limit [here](/data-platform/accelerate/limitations#response-size-limit).
```

Comment:
how about a different formatting. the "titles" you've written in bold seem to be "code" as well (I assume so because of the PascalCasing, so I think they should be written in backticks as well). also I think it makes sense to include them in the section heading already?! 

```suggestion
#### <inlinecode>P6004</inlinecode> (<inlinecode>QueryTimeout</inlinecode>)

The global timeout of Accelerate has been exceeded. You can find the limit [here](/data-platform/accelerate/limitations#query-timeout-limit).

#### <inlinecode>P6005</inlinecode> (<inlinecode>InvalidParameters</inlinecode>)

The user supplied invalid parameters. Currently only relevant for transaction methods. For example, setting a timeout that is too high. You can find the limit [here](/data-platform/accelerate/limitations#interactive-transactions-query-timeout-limit).

#### <inlinecode>P6009</inlinecode> (<inlinecode>ResponseSizeLimitExceeded</inlinecode>)

The global response size limit of Accelerate has been exceeded. You can find the limit [here](/data-platform/accelerate/limitations#response-size-limit).
```